KIPs/kip-82.md

```diff
@@ -0,0 +1,376 @@
+---
+kip: 82
+title: A new GC reward structure due to abolition of the Gini coefficient
+author: Yeri<yeriel.lee@krustuniverse.com>, Daniel<daniel.cc@krustuniverse.com>, Aidan<aidan.kwon@krustuniverse.com>, Ollie<ollie.j@krustuniverse.com>, Sam<sam.seo@krustuniverse.com>, Uno<uno.lee@krustuniverse.com>, Eddie<eddie.kim0@krustuniverse.com>
+discussion-to: https://govforum.klaytn.foundation/t/proposal-a-new-gc-reward-structure-to-abolition-of-the-gini-coefficient/21
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-09-21
+---
+
+
+## Simple Summary
+A renewal of the Governance Council(GC) reward structure due to abolition of the Gini coefficient.
+
+## Abstract
+In addition to traditional enterprises, Klaytn is expanding the Governance Council (GC) by bringing DAOs in response to the growth of nontraditional entities. Such a change in the GC ecosystem will restructure the entire Klaytn governance structure. Abolishing the current block proposal selection method based on the Gini coefficient and staking quantity, the new reward program alleviates the problem of providing insufficient compensation for the network contribution and block verification due to lack of staking. The instability of certain nodes with high staking amounts can result in poor network stability as a whole. The renewed structure provides 20% of GC rewards to block proposers and the rest to the stakers. This form is to reward the participants for providing network stability with node operation and financial contribution.
+
+## Motivation
+Klaytn has been compensating GC with block creation rewards based on the quantity of staking and the Gini coefficient. This included rewards incurred by minting and gas fee rewards generated by transactions during block creation. As mentioned in the [Klaytn 2.0 Lightpaper](https://klaytn.foundation/wp-content/uploads/Lightpaper.pdf), Klaytn is abolishing the Gini coefficient since this measure demotivates members from staking. Discontinuation of the Gini coefficient increases the risk to network stability due to the expanding reliance on certain nodes. This change ultimately calls for the revision of block proposer selection and reward structure. In an effort to enhance individual voting power and promote ultimate decentralization, we aim to accomplish the following changes:
```

Comment:
```suggestion
Klaytn has been compensating GC with block proposal rewards based on the quantity of staking and the Gini coefficient. This included rewards incurred by minting and gas fee rewards generated by transactions during block proposal. As mentioned in the [Klaytn 2.0 Lightpaper](https://klaytn.foundation/wp-content/uploads/Lightpaper.pdf), Klaytn is abolishing the Gini coefficient since this measure demotivates members from staking. Discontinuation of the Gini coefficient increases the risk to network stability due to the expanding reliance on certain nodes. This change ultimately calls for the revision of block proposer selection and reward structure. In an effort to enhance individual voting power and promote ultimate decentralization, we aim to accomplish the following changes:
```